hibernate-orm-panache-resteasy/src/test/java/FruitsEndpointTest.java

```diff
@@ -2,12 +2,31 @@
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.core.IsNot.not;
 
+import com.github.dockerjava.api.model.ExposedPort;
+import com.github.dockerjava.api.model.PortBinding;
+import com.github.dockerjava.api.model.Ports;
 import io.quarkus.test.junit.QuarkusTest;
 import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
 
 @QuarkusTest
+@Testcontainers
 public class FruitsEndpointTest {
 
+    @Container
+    static PostgreSQLContainer DATABASE = new PostgreSQLContainer<>("postgres:10.5")
```

Comment:
Why not https://www.testcontainers.org/modules/databases/#database-containers-launched-via-jdbc-url-scheme ?:)

1. can be used as a static value somewhere in the test config
2. randomizes the ports
3. Much smaller change

Reply:
Although I agree that in a real project this would be interesting, as this is a quickstart I am not sure to add this on the JDBC Url. Basically, because of anyone who wants to start with Quarkus will also need to understand what is this `tc` thing part in URL. And should know that it is nothing related to Quarkus but for testing purposes. So since these quickstarters are used as tutorials, I'd advocate to not mix things.

Reply:
Saying that I like so much this feature.

Reply:
I see. Solid argument, agreed 👍